improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -745,42 +756,40 @@ def _mean_and_variance_to_percentiles(
             ValueError: If any of the resulting percentile values are
                 nans and these nans are not caused by a zero variance.
         """
-        calibrated_forecast_predictor_data = (
-            calibrated_forecast_predictor.data.flatten())
-        calibrated_forecast_variance_data = (
-            calibrated_forecast_variance.data.flatten())
+        location_parameter_data = location_parameter.data.flatten()
+        scale_parameter_data = scale_parameter.data.flatten()
 
         # Convert percentiles into fractions.
         percentiles = np.array(
             [x/100.0 for x in percentiles], dtype=np.float32)
 
         result = np.zeros((len(percentiles),
-                           calibrated_forecast_predictor_data.shape[0]),
-                          dtype=np.float32)
+                           location_parameter_data.shape[0]), dtype=np.float32)
 
         self._rescale_shape_parameters(
-            calibrated_forecast_predictor_data,
-            np.sqrt(calibrated_forecast_variance_data))
+            location_parameter_data,
+            np.sqrt(scale_parameter_data))
 
         percentile_method = self.distribution(
             *self.shape_parameters,
-            loc=calibrated_forecast_predictor_data,
-            scale=np.sqrt(calibrated_forecast_variance_data))
+            loc=location_parameter_data,
+            scale=np.sqrt(scale_parameter_data))
 
         # Loop over percentiles, and use the specified distribution with the
-        # mean and variance to calculate the values at each percentile.
+        # location and scale parameter to calculate the values at each
+        # percentile.
```

Comment:
Specified distribution? Where do we use that?

Reply:
I've made a minor update to this sentence. Is that clearer?